runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -132,14 +133,73 @@ private void drawSceneOverlays(Graphics2D graphics, Player actor, PlayerRelation
 			OverlayUtil.renderPolygon(graphics, actor.getConvexHull(), color);
 		}
 
-		if (Arrays.asList(plugin.getLocationHashMap().get(relation)).contains(PlayerIndicationLocation.TILE))
+		if (Arrays.asList(plugin.getLocationHashMap()
+			.getOrDefault(relation, new Object[]{null}))
```

Comment:
apply this to the if statement above too, and make the Object array a `private static final`